From c67879e404685086f351e1fffa4b43f73317ee09 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sat, 2 Feb 2019 11:16:38 +0800
Subject: [PATCH 4/4] USB: common: fix misleadingly indented in
 of_usb_get_maximum_speed()

To avoid below warning:
drivers/usb/common/common.c:151:3: note: ...this statement, but the latter is
misleadingly indented as if it were guarded by the 'for'
   return USB_SPEED_UNKNOWN;
   ^~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/common/common.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/common/common.c b/drivers/usb/common/common.c
index 81fe02f..df265c9 100644
--- a/drivers/usb/common/common.c
+++ b/drivers/usb/common/common.c
@@ -148,7 +148,7 @@ enum usb_device_speed of_usb_get_maximum_speed(struct device_node *np)
 		if (strcmp(maximum_speed, speed_names[i]) == 0)
 			return i;
 
-		return USB_SPEED_UNKNOWN;
+	return USB_SPEED_UNKNOWN;
 }
 EXPORT_SYMBOL_GPL(of_usb_get_maximum_speed);
 
-- 
1.7.9.5

